tests/common/devices/multi_asic.py

```diff
@@ -602,8 +608,8 @@ def docker_copy_to_all_asics(self, container_name, src, dst):
         """This function copy from host to ALL asics"""
         duthost = self.sonichost
         if duthost.is_multi_asic:
```

Comment:
same as above , container name to be appended with asic index only for asic services.
if duthost.is_multi_asic and container_name in [DEFAULT_ASIC_SERVICE](https://github.com/Azure/sonic-mgmt/blob/2e10e91da0c12735427805c06b5e540a24ade79c/tests/common/devices/sonic.py#L35)